packages/component-library/styles/type.scss

```diff
@@ -11,30 +11,30 @@ $ca-weight-medium: 500;
 $ca-weight-semibold: 500; // Note: in Sketch, semibold is 600. But murmur has an existing value of semibold=500 that is heavily used.
 
 // Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
-$ca-default-font-family: "Open Sans", Helvetica, Arial, sans-serif;
-$ca-default-font-base-size: 0.875rem; /* 14px */
-$ca-default-font-descender-height: 0.115;
+$ca-inter-font-family: "Inter";
```

Comment:
This is the variable that's used globally in other repos, right? Should It have some fallback?

Reply:
What I realised is that none of the other fonts (including Inter) have fallbacks 😬 , so I kind of wanted to defer that. I mean it seems pretty harmless to add fallbacks, but I'm not 100% sure what the current fallback best practice is. I'll add a github issue about this. Now that I look at it, they kz-xxxx mixins have the fallbacks, just not the legacy `$ca-inter-font-family`. I'm just not sure of the ramifactions of changing that. I guess now is a good time though 🤷‍♂️

Reply:
I also wonder if Helvetica is a better fallback than just `sans-serif`, because the system font in Apple products is no longer Helvetica. I think it's "SF Pro"?

Reply:
That's not ideal 😬 At the very least it should be `Inter, sans-serif` (with the view to come back to this later). I'm not sure what a better fallback is at this point in time.